### DISCUSSION TITLE
LAN Modbus Not Working???

### DISCUSSION THREAD

**Discussion by smoketabs** [StealthChesnut/HA-FoxESS-Modbus#80] [Q&A]
Hi.

Noticed the statement on the home page: "22/12/22 - Manager version 1.56 appears to break the LAN connectivity option!".

Does this mean LAN modbus doesn't work? I ask as I don't have a FoxESS yet and am keen to know if I will be forced down API for sensor information?

Thanks in advance and apologies if it's a stupid question.

Cheers

**Comment by tagware** [comment]
Hi

I Use the LAN port and it does seem to be disabled at 1.56v Manager.

I have yet to be informed by Fox if this was by intent or cock up.🤔

As my system is all updated except the Manager and is stable. I haven't
found a reason to upgrade to 1.56.

However, I have the battery 🔋 not hyp. So, there might be a good reason to
upgrade.

Hope this helps.

**Reply by bkortleven** [reply]
I have the 1.56 update installed/pushed/ (didn't check if it was delivered with this version installed in september 2022?) and on the 23rd of January, all of a sudden all ethernet traffic stopped. No stats or DHCP anymore, just status/100Mb-HD led on the switch side... Started just at/after midnight... I am in no way capable to resurrect the ethernet port/data anymore. Restarted the whole inverter after full shutdown, other port, other config, no DHCP but fixed IP configured... no resolution...
Anyone any idea? Is there a possibility to roll-back the comm firmware? Where can we find the files to up/downgrade ourselves?
